haskell-style.md

```diff
@@ -156,6 +172,88 @@ instance FromJSON (GameSession Never MathStandardAssignmentId) where
         <*> pure Never
 ```
 
+```diff
+ instance FromJSON (GameSession Never MathStandardAssignmentId) where
+   parseJSON =
+     withObject "cannot parse GameSession" $ \o ->
+-      GameSession
++      GameSession2
```

Comment:
Seems like we should figure out a consensus on this style point. I'm totally down to get rid of this and prefer no newline.

Reply:
Yeah, me too. I think it's a broader discussion though, to be had separately from operator-first. For example, I know @cdparks 

```hs
uses =
  newlines
    $ like mad
```

And I'm happy to let him, while I don't prefer it myself.

So, I'd be happy to not explicitly mention it here (so revert 6d64f2d, basically) and pick up that discussion separately.

I'm also down to leave this ambiguous for now, and bank on Brittany making it [moo](https://assets-auto.rbl.ms/e918f7c316ce8442edba7a0c9e58c62fe0d3b4996eb6b1e04d3f2c6b0481289c).

Reply:
:laughing: moo :rofl:

Reply:
```haskell
this =
  is
    a
    fair
    characterization
```